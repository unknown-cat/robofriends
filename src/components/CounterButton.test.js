import React from "react";
import {shallow} from "enzyme";
import CounterButton from "./CounterButton";

it('expects to render CounterButton component', () => {
	expect(shallow(<CounterButton/>)).toMatchSnapshot()
})