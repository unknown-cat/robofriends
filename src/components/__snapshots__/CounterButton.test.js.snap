// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`expects to render CounterButton component 1`] = `
<button
  onClick={[Function]}
>
  Count: 
  1
</button>
`;
